Assets/Scripts/UI/Menu's/PauseController.cs

```diff
@@ -38,7 +38,8 @@ public void TogglePauseMenu(bool isPaused)
         pauseScreen.SetActive(!isPaused);
         _isPaused = !isPaused;
 
-        Time.timeScale = isPaused ? 1 : 0;
+        if(!isPaused) TimeScaleController.SetTimeScaleOnInactive();
+        else TimeScaleController.SetTimeScaleOnActive();
```

Comment:
```suggestion
        Time.timeScale = SetTimeScale(isPaused ? 1 : 0);
```
Als je de feedback hier boven doet dan kan je dit zo typen.